### ISSUE TITLE
Compatibility with Sand Piles added by Immersive Weathering

### ISSUE THREAD

**Issue by Chribit** [TheDeathlyCow/scorchful#63]
**Is your feature request related to a problem? Please describe.**
I'm currently putting together a nice little mod pack for me and my friends and amongst others, I've added "Immersive Weathering" and "Scorchful". Both of these mods add sand layers - or "sandy" versions of snow layers. Both look identical when placed, the only difference being the item form. They obviously can't be stacked on top of each other, but both are generated in the world one way or the other. This leads to confusion for players and also potentially unappealing visuals in the desert since some sand piles seam to accumulate during sand storms but others don't.

**Describe the solution you'd like**
I think this could be solved by checking for Immersive Weathering usage alongside Scorchful and then resolving this conflict somehow. Maybe utilise Immersive Weathering's sand piles instead of Scorchful's if available? I'd be happy to post a similar issue on Immersive Weathering's issue page if you want. Just would be sick if these two mods were fully compatible mechanics wise.

**Comment by TheDeathlyCow** [comment]
This sounds like something that could be implemented in [Thermoo Patches](https://github.com/TheDeathlyCow/thermoo-patches)

**Comment by Chribit** [comment]
Oh yes absolutely! I wasn't aware that mod existed for some reason - gotta install that as well since i'm also using fabric seasons and ad astra :D

Should i repost this issue to that repo?

**Comment by TheDeathlyCow** [comment]
Yeah I think this would be a good issue to open up over there, not here.

**Comment by Chribit** [comment]
reposted in Thermooo Patches repo:
https://github.com/TheDeathlyCow/thermoo-patches/issues/7

I guess this can be closed?

**Comment by TheDeathlyCow** [comment]
thanks. yeah i guess this can be closed. though i think i'll still need to make some changes to scorchful too to make this a little less painful to implement.
